modules/os2forms_digital_signature/src/Form/SettingsForm.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Drupal\os2forms_digital_signature\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * Digital post settings form.
+ */
+class SettingsForm extends ConfigFormBase {
+  use StringTranslationTrait;
+
+  /**
+   * Name of the config.
+   *
+   * @var string
+   */
+  public static $configName = 'os2forms_digital_signature.settings';
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'os2forms_digital_signature_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [self::$configName];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form['os2forms_digital_signature_remove_service_url'] = [
```

Comment:
I think we have a typo here ("remove" should be "remote"):

```suggestion
    $form['os2forms_digital_signature_remote_service_url'] = [
```